app/src/main/java/org/greenstand/android/TreeTracker/database/DbHelper.kt

```diff
@@ -80,9 +82,9 @@ class DbHelper(private val myContext: Context, name: String, factory: CursorFact
                 while (treesCursor.moveToNext()) {
 
 
-                    val sharedPreferences = myContext.getSharedPreferences(
+                    val sharedPreferences1 = myContext.getSharedPreferences(

```

Comment:
i don't understand the rename , if there is another variable with the same name please use a different meaningful name , i.e. userSharedPrefrences